src/main/java/com/uber/jenkins/phabricator/PhabricatorNotifierDescriptor.java

```diff
@@ -103,4 +104,12 @@ public String getUberallsURL() {
     public void setUberallsURL(String value) {
         uberallsURL = value;
     }
+
+    public boolean getBlueOceanURI() {
```

Comment:
The methods and field make it appear like the property is a URI and not a boolean. Please change the property name